charts/localstack/templates/ingress.yaml

```diff
@@ -27,14 +27,14 @@ spec:
   {{- range .Values.ingress.tls }}
     - hosts:
       {{- range .hosts }}
-        - {{ . | quote }}
+        - {{ tpl . $ | quote }}
```

Comment:
When testing this with fixed `test-values.yaml` (see other comment) we can see that this value is also used in the deployment notice:
```
helm install ls-helm-demo ./charts/localstack/ -f ./charts/localstack/test-values.yaml                                                                                                                            11:30:12
NAME: ls-helm-demo
LAST DEPLOYED: Tue May 28 11:30:13 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1
NOTES:
1. Get the application URL by running these commands:
  http://{{ .Values.ingressHost }}/
```

Please make sure to adjust all usages of the ingress hosts and pipe them through the `tpl` function everywhere.
This specific case is here: https://github.com/localstack/helm-charts/blob/8308a208bde36d6d4160e33c68522cdc2a2a7521/charts/localstack/templates/NOTES.txt#L2-L7